bin/build/src/i18n/common.clj

```diff
@@ -17,7 +17,8 @@
    (sorted-set)
    (for [^java.io.File file (.listFiles (io/file (u/filename u/project-root-directory "locales")))
          :let               [file-name (.getName file)]
-         :when              (str/ends-with? file-name ".po")]
+         :when              (and (str/ends-with? file-name ".po")
+                                 (not (str/starts-with? file-name "metabase")))]
```

Comment:
@metabase/core-backend-admin-webapp checkout my mad clojure skillz

Reply:
Skills are suspiciously good, but I don't get the logic, why skip `metabase*`?

Reply:
because there's not a `metabase` language yet!

previously, the source file was called `metabase.pot`, but I had to change it to `metabase.po` which made metabase think that file was a translation file instead of a string source file

Reply:
Oh right! 🤦‍♂️